index.js

```diff
@@ -318,7 +318,8 @@ ${inject.body || ''}
           '-preset', ffmpegOptions.preset,
           '-crf', ffmpegOptions.crf,
           '-movflags', 'faststart',
-          '-pix_fmt', 'yuv420p'
+          '-pix_fmt', 'yuv420p',
+          '-r', fps
```

Comment:
this argument already exists earlier in the args https://github.com/ed-asriyan/puppeteer-lottie/blob/patch-1/index.js#L313

Reply:
Yes, but we need the second **-r**